README.md

```diff
@@ -1211,6 +1211,31 @@ we can do two things:
 1. List all distinct binary packages that should be promoted.  Often a source package will have binary packages that aren't actually needed in main.  Things like `-doc`, `-autopilot` or `-dbgsym`.  These can stay in universe, and it is a kindness to list only the packages we need for the archive team member that does the promotion.
 1. Recommend the owning team to add their corresponding team bug subscriber during the MIR process.
 
+## MIR Review - service level objective
+
+MIR reviews take time, scaling up with the complexity of the case. Sadly it is usually
+the very complex, very special cases that come in very late then are not happy with the
+velocity of review.
+
+While we aim for more when possible, our goal is to assign (in the weekly meeting)
+one review per active MIR member per week, and coming back with the result until
+the next meeting or earlier. That usually means we can handle ~4 cases per week,
+sometimes more (if we can make the space or they are trivial) and sometimes
+less (PTO times).
+
+So far this used to be always enough and allowed <week responses for almost
```

Comment:
Should probably read "< 1 week" for improved clarity.

Reply:
ack, done